packages/core/src/system.ts

```diff
@@ -34,6 +46,36 @@ export function createSystem<T extends ActorSystemInfo>(): ActorSystem<T> {
 
       keyedActors.set(systemId, actorRef);
       reverseKeyedActors.set(actorRef, systemId);
+    },
+    inspect: (observer) => {
+      observers.add(observer);
+    },
+    _sendInspectionEvent: (event) => {
+      const resolvedInspectionEvent: ResolvedInspectionEvent = {
+        id: Math.random().toString(),
+        createdAt: new Date().toString(),
+        ...event,
+        systemId: system.root.sessionId
```

Comment:
I find this a little confusing. It is called systemId, but is populated with the root sessionId. 

I understand that this is meant to identify the system within which this is happening, but the `systemId` prop is already used in the spawn and invoke options. A child machine can have both sessionId and systemId, which may have been assigned to it when it was spawned/invoked.
I guess, I'm trying to say that Actor.systemId has a different meaning than InspectionEvent.systemId. 

- Actor.systemId = a key in the system Map,
- InspectionEvent.systemId = sessionId of the system itself.

Reply:
Yeah I struggled with this too, maybe we should rename this to `rootSessionId`

Reply:
Yes, I think rootSessionId is a good name.